.github/workflows/release.yaml

```diff
@@ -0,0 +1,39 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
+    tags-ignore:
+      - '**'
+    paths-ignore:
+      - '*.md'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
```

Comment:
When we login to Dockerhub why do we login to ghcr?

Reply:
I was just pushing to both, I'll remove ghcr if all we want is dockerhub

Reply:
I'd say docker hub alone is fine.